### PR TITLE
PEP8 fixes

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Few pep8 fixes for recent commits.

### CODE REVIEW

File: blivet/util.py [hunk 1]
```diff
@@ -1079,7 +1080,9 @@ class EvalMode(Enum):
 
 
 class DependencyGuard(object, metaclass=abc.ABCMeta):
+
```
Comment (vpodzime) [inline blivet/util.py]: Hmm a docstring probably belongs here. I'll add it.

**Comment by vpodzime** [comment]
Looks good to me.

**Comment by vojtechtrefny** [comment]
Added new commit to make sure `make pep8` is executed even if `make pylint` fails.

**Comment by vpodzime** [comment]
Looks good to me.

**Comment by vojtechtrefny** [comment]
Update: ignore "E402 module level import not at top of file"

**Comment by vpodzime** [comment]
Still looks good to me.

**Comment by dwlehman** [comment]
You didn't want to fix the other ones while you're at it?

```
python3-pep8 --ignore=E501,E402 blivet/ tests/ examples/
blivet/__init__.py:48:1: E731 do not assign a lambda expression, use a def
blivet/__init__.py:74:1: E731 do not assign a lambda expression, use a def
blivet/i18n.py:44:1: E731 do not assign a lambda expression, use a def
blivet/i18n.py:51:5: E731 do not assign a lambda expression, use a def
blivet/i18n.py:52:5: E731 do not assign a lambda expression, use a def
blivet/i18n.py:54:5: E731 do not assign a lambda expression, use a def
blivet/i18n.py:55:5: E731 do not assign a lambda expression, use a def
blivet/devicelibs/edd.py:367:44: W503 line break before binary operator
blivet/devices/lvm.py:721:13: E731 do not assign a lambda expression, use a def
blivet/devices/lvm.py:740:13: E731 do not assign a lambda expression, use a def
```

**Comment by vojtechtrefny** [comment]
Ok, everything should be fixed now. (At least jenkins is happy, my older `pep8` doesn't warn about these at all).

**Comment by dwlehman** [comment]
Looks good.

**Pull request merged**